src/LsmTree.hs

```diff
@@ -33,14 +34,17 @@ insertTree x tree = case tree of
         False -> TreeNode l val (insertTree x r)
 
 -- | Data type that stores values in memory.
-data Table a = Table [a]
-    deriving stock (Eq, Show)
+data Table = Table
+    { fileName :: String
+    , size :: Int
```

Comment:
What is this size? Current size of a file? Or a limit? This field should be documented :slightly_smiling_face: